pkg/client/v1/model/volume.go

```diff
@@ -4,182 +4,478 @@
-	FolderName string `json:"folder_name,omitempty"`
-	// CachingEnabled
-	CachingEnabled bool `json:"caching_enabled,omitempty"`
-	// PreviouslyDeduped
-	PreviouslyDeduped bool `json:"previously_deduped,omitempty"`
-	// DedupeEnabled
-	DedupeEnabled bool `json:"dedupe_enabled,omitempty"`
-	// VpdT10
-	VpdT10 string `json:"vpd_t10,omitempty"`
-	// VpdIeee0
-	VpdIeee0 string `json:"vpd_ieee0,omitempty"`
-	// VpdIeee1
-	VpdIeee1 string `json:"vpd_ieee1,omitempty"`
-	// AppCategory
-	AppCategory string `json:"app_category,omitempty"`
-	// LimitIops
-	LimitIops int32 `json:"limit_iops,omitempty"`
-	// LimitMbps
-	LimitMbps int32 `json:"limit_mbps,omitempty"`
-	// NeedsContentRepl
-	NeedsContentRepl bool `json:"needs_content_repl,omitempty"`
-	// ContentReplErrorsFound
-	ContentReplErrorsFound bool `json:"content_repl_errors_found,omitempty"`
-	// LastContentSnapBrCgUID
-	LastContentSnapBrCgUID float64 `json:"last_content_snap_br_cg_uid,omitempty"`
-	// LastContentSnapBrGID
-	LastContentSnapBrGID float64 `json:"last_content_snap_br_gid,omitempty"`
-	// LastContentSnapID
-	LastContentSnapID float64 `json:"last_content_snap_id,omitempty"`
-	// CksumLastVerified
-	CksumLastVerified float64 `json:"cksum_last_verified,omitempty"`
-	// PreFilter
-	PreFilter string `json:"pre_filter,omitempty"`
-	// SrepLastSync
-	SrepLastSync float64 `json:"srep_last_sync,omitempty"`
-	// SrepResyncPercent
-	SrepResyncPercent float64 `json:"srep_resync_percent,omitempty"`
+   
+    // Identifier for the volume.
+    
+ 	ID *string `json:"id,omitempty"`
+   
+    // Name of the volume.
+    
+ 	Name *string `json:"name,omitempty"`
+   
+    // Fully qualified name of volume.
+    
+ 	FullName *string `json:"full_name,omitempty"`
+   
+    // Name of volume used for object search.
+    
+ 	SearchName *string `json:"search_name,omitempty"`
+   
+    // Volume size in mebibytes. Size is required for creating a volume but not for cloning an existing volume.
+    
+   	Size *int64 `json:"size,omitempty"`
+   
+    // Text description of volume.
+    
+ 	Description *string `json:"description,omitempty"`
+   
+    // Name of performance policy.
+    
+ 	PerfpolicyName *string `json:"perfpolicy_name,omitempty"`
+   
+    // Identifier of the performance policy. After creating a volume, performance policy for the volume can only be changed to another performance policy with same block size.
+    
+ 	PerfpolicyID *string `json:"perfpolicy_id,omitempty"`
+   
+    // Amount of space to reserve for this volume as a percentage of volume size.
+    
+   	Reserve *int64 `json:"reserve,omitempty"`
+   
+    // This attribute is deprecated. Alert threshold for the volume's mapped usage, expressed as a percentage of the volume's size. When the volume's mapped usage exceeds warn_level, the array issues an alert. If this option is not specified, array default volume warn level setting is used to decide the warning level for this volume.
+    
+   	WarnLevel *int64 `json:"warn_level,omitempty"`
```

Comment:
Note that golang fmt requires the name of the field to be the first part of the comment...

// WarnLevel - <something>
WarnLevel *int64

Reply:
Sure will fix it. thanks